ios/Sources/Component/SearchController.swift

```diff
@@ -1,33 +1,40 @@
+import Styleguide
 import UIKit
 
 @propertyWrapper
 public class SearchController: NSObject {
 
     public let wrappedValue = UISearchController()
     private let searchTextDidChangeTo: (String) -> Void
-    private let willDismissSearchController: () -> Void
+    private let isSearchTextEditing: (Bool) -> Void
```

Comment:
I think this may not be correct naming for closure.

`**DidChange` may be better.